src/rlookup.c

```diff
@@ -443,12 +466,9 @@ static int getKeyCommonJSON(const RLookupKey *kk, RLookupRow *dst, RLookupLoadOp
       return REDISMODULE_OK;
     }
   } else {
-    RedisJSON jsonValue = japi->next(jsonIter);
-    japi->freeIter(jsonIter);
```

Comment:
How come we don't need to free?

Reply:
The iterator is freed in `jsonIterToValue`